iphone/Maps/Classes/CustomAlert/DownloadTransitMapsAlert/MWMDownloadTransitMapAlert.mm

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithIndexes:(storage::TCountriesVec const &)countriesVec isM
   if (self)
   {
     NSMutableArray * titles = [@[] mutableCopy];
-    size_t totalRoutingSize = 0;
+    storage::TMwmSize totalRoutingSize = 0;
```

Comment:
Не вижу ни одной причины, зачем пришлось бы менять размер. Увеличивать дальше? Таких устройств не существует. А код читабельный важен уже сейчас.

> On Apr 6, 2016, at 15:53, Vladimir Byko-Ianko <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomAlert/DownloadTransitMapsAlert/MWMDownloadTransitMapAlert.mm <https://github.com/mapsme/omim/pull/2785#discussion_r58698733>:
> 
> > @@ -27,7 +27,7 @@ - (instancetype)initWithIndexes:(storage::TCountriesVec const &)countriesVec isM
> >    if (self)
> >    {
> >      NSMutableArray * titles = [@[] mutableCopy];
> > -    size_t totalRoutingSize = 0;
> > +    storage::TMwmSize totalRoutingSize = 0;
> Эту штуку пришлось заменять. Вероятно, это будет нужно сделать еще раз. Думаю, с typedef по спокойнее.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2785/files/bb6f82e1bd96a8244a5f19432fef96d0dcdfa806#r58698733>